### PR TITLE
Home work 19

### PR DESCRIPTION
**Pull request by k1ddd**

### CODE REVIEW

File: burger/src/components/Main/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  return (
+    <div className="main">
+      <div className="left__aside">
+        <div>
+          <h2>Our Price</h2>
+          <p>Bacon: 0.75 UAH</p>
+          <p>Cheese: 1.7 UAH</p>
+          <p>Salad: 0.95 UAH</p>
+          <p>Pickle: 1.25 UAH</p>
+          <p>Meat: 2 UAH</p>
```
Comment (roskyba) [inline burger/src/components/Main/index.js]: Better to save price in an array. It is much easier to maintain and update them later

Comment (roskyba) [reply burger/src/components/Main/index.js]: And use components, for example, `<Prices />`

File: burger/src/components/Main/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            <button className="button">-</button>
+            <img src={pickles} className="ingredients" alt="Logo" />
+          </div>
+          <div className="btn__choice">
+            <button className="button">+</button>
+            <span className="amount">0 </span>
+            <button className="button">-</button>
+            <img src={meat} className="ingredients" alt="Logo" />
+          </div>
+        </div>
```
Comment (roskyba) [inline burger/src/components/Main/index.js]: You must break down this huge UI into a few components. Easier to reason and debug

Comment (roskyba) [reply burger/src/components/Main/index.js]: The main reason you are using React - you can create components and use them as building blocks. In much easier and elegant way

In your case - you have 1 component that is huge and easy to get lost. You have to refactor

**Review by roskyba — APPROVED** [review]
Good work!

File: burger/src/components/Main/index.js (hunk 2 above)
Comment (k1ddd) [reply burger/src/components/Main/index.js]: fix it

**Pull request merged**